lib/modem_info/modem_info.c

```diff
@@ -873,7 +873,7 @@ int modem_info_get_rsrp(int *val)
 		return -ENOENT;
 	}
 
-	*val = *val - RSRP_OFFSET_VAL;
+	*val = RSRP_IDX_TO_DBM(*val);
```

Comment:
This applies to other places where the macros are used as well, but the value returned by the macro is a float, and it's assigned into an int, so probably it should be rounded appropriately.

Reply:
Maybe. As discussed this is how it's been and after all, this doesn't really affect on any functionality.